ecgtools/parsers/cesm.py

```diff
@@ -44,6 +45,21 @@
     'ww3.h': {'component': 'wav'},
 }
 
+frequency_dict = {
+    'cam.h0': 'monthly',
+    'cam.h1': 'daily',
+    'cam.h2': 'hourly6',
+    'cice.h': 'monthly',
+    'cice.h1': 'daily',
+    'clm2.h0': 'monthly',
+    'clm2.h1': 'daily',
+    'pop.h.ecosys.nyear1': 'annual',
+    'pop.h.ecosys.nday1': 'daily',
+    'pop.h': 'monthly',
+    'rtm.h0': 'monthly',
+    'rtm.h1': 'daily',
```

Comment:
Could we make these consistent with what CESM produces i.e. as far I can remember `monthly` is recorded as `month_1` in CESM2 output?